### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by amortensen1229**
updated readme with newest endpoints and switched database description to include local setup. Also included ER diagram.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,8 +24,12 @@ uvicorn main:app --reload
 ## **Active Endpoints**
 1. ```POST /menu```
     - Creates menu object and stores in database instance.
-2. ```More endpoints coming soon```
-
+2. ```POST /rest_database```
```
Comment (norbusonam) [inline README.md:27]: Should be reset_database

**Review by norbusonam — COMMENTED** [review]
we should probably remove and .gitignore .DS_Store

File: README.md (hunk 1 above)
Comment (amortensen1229) [reply README.md:27]: good catch let me fix and create a new PR

**Comment by amortensen1229** [comment]
@norbusonam fixed!

**Review by norbusonam — APPROVED** [review]

**Pull request merged**